JavaScriptTraining/JsTask1_3.js

```diff
@@ -0,0 +1,29 @@
+const list_title = `
+======================
+現在持っているタスクの一覧
+======================`;
+
+const type_message =
```

Comment:
Javascript の世界では  `snake_case` より `camelCase` の方が一般的です。

```suggestion
const typeMessage =
```